python/setup.py

```diff
@@ -2,7 +2,7 @@
 setup(
     name = 'pytispark',
     packages = ['pytispark'],
-    version = '0.1.3',
+    version = '1.0.0-RC1',
```

Comment:
should this be `1.0.0-RC1` or `1.0-RC1`?

Reply:
I saw both on [Pypi](https://pypi.python.org)

Reply:
The former could do more fined grained version control.

Reply:
But we generally do not modify this module...